packages/yourloops/lib/auth/models/country.model.ts

```diff
@@ -39,5 +39,7 @@ export enum CountryCodes {
   Netherlands = 'NL',
   Spain = 'SP',
   Switzerland = 'CH',
+  UnitedKingdom = 'UK',
+  Japan = 'JP',
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** Alphabetical order for JP value 👼